util/types/mydecimal.go

```diff
@@ -41,6 +44,13 @@ const (
 
 	MaxFraction = 30
 	DivFracIncr = 4
+
+	// AlwaysRound rounds regardless of the decimal.
+	AlwaysRound RoundMode = 0
+	// NormalRound rounds normally.
+	NormalRound RoundMode = 5
+	// NeverRound just truncates the decimal.
+	NeverRound RoundMode = 9
```

Comment:
The code https://github.com/mysql/mysql-server/blob/71f48ab393bce80a59e5a2e498cd1f46f6b43f9a/strings/decimal.c#L1655 shows that 10 represents truncate, I'll take a look.